at_demo_data/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 2.0.0
+- BREAKING CHANGE: replaced unicode U+1F6E0 with U+1F6E0 U+FE0F for all
+  :hammer_and_wrench: emojis  
+  (normalized the emojis by adding the variant selector)
+
+- The reason for making this change is that shells (e.g. bash, zsh) and emoji 
+  normalization techniques expect the variant selector to be there. Specifically
+  with the :hammer_and_wrench: emoji. Most shells recognize the U+1F6E0
+  character as a double wide character which takes up a single column. The 
+  U+FE0F is a zero-width character which also takes up a single column. Thus
+  correcting the number of columns to match the width of the characters. Without
+  this normalization, it becomes difficult to use the characters over the shell.
```

Comment:
@cpswan here is the explainer of why I did this change. I appended U+FE0F (variant selector) to every U+1F6E0 (hammer and wrench emoji) where it was missing, thus normalizing the emojis. This variant selector is automatically added by emoji pickers on most (if not all) major platforms, hence why we have had to copy paste in the past.